app/models/container.rb

```diff
@@ -4,6 +4,7 @@ class Container < ActiveRecord::Base
 
   has_one    :container_group, :through => :container_definition
   delegate   :ext_management_system, :to => :container_group
+  delegate   :container_replicator, :to => :container_group
```

Comment:
@simon3z, I think `:to` is the correct syntax here for `delegate`.

http://api.rubyonrails.org/classes/Module.html#method-i-delegate